### ISSUE TITLE
test case error - cannot read property 'querySelectorAll'

### ISSUE THREAD

**Issue by sathishkumar-thangavel** [akiran/react-slick#1830]
### Issue causing release Tag: '0.26.1'

This line causing the issue
https://github.com/akiran/react-slick/blame/bb11613a3c615da499ed3ca8eafdff029902b8f6/src/inner-slider.js#L294

```
import React from 'react';
import renderer from 'react-test-renderer';
import Slider from 'react-slick';


describe('Testimonial carousel', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Slider />).toJSON();
    expect(tree).toBeTruthy();
  });
});
```

**Comment by sathishkumar-thangavel** [comment]
###  _**Temporary solution is using 0.25.0**_

**Comment by jvitela** [comment]
Same problem here with version 0.27.3

**Comment by alieslamifard** [comment]
Same problem with `0.27.13`

**Comment by akiran** [comment]
Fixed it in react-slick@0.27.14

**Comment by gerisztein** [comment]
@akiran for me it's quite the opposite, it was working before and 0.27.14 broke it 😢
